docs/source/gbd2017_models/risks/risk_exposures_gbd.rst

```diff
@@ -0,0 +1,15 @@
+.. _2017_risk_exposures_gbd:
+
+=====================
+Risk Exposures in GBD
+=====================
+
+..	todo::
+	This page will follow the analogous vivarium_research\docs\source\gbd2017_models\causes\causes.rst page.

```

Comment:
I think both of these pages ("Risk Exposures in GBD" and "Cause Models in GBD") are in the wrong folders. I think they should both go in the "Understanding GBD Data" section, i.e. https://vivarium-research.readthedocs.io/en/latest/data.html  (`vivarium_research/docs/build/html/data.html` - we'll have to create a new folder there).  What do other people think?

Reply:
I think probably yes, but i'll leave that for a different PR.

Reply:
Actually, I think definitely yes. Still leaving for a different PR.